packages/datepicker/datepicker.scss

```diff
@@ -1,41 +1,98 @@
 @import "~@fremtind/jkl-core/variables/_all.scss";
+@import "~@fremtind/jkl-core/mixins/_all.scss";
 @import "~@fremtind/jkl-core/_functions.scss";
 @import "~@fremtind/jkl-text-input/text-input.scss";
 @import "~@fremtind/jkl-dropdown/dropdown.scss";
 
-$date-width: rem(46px);
+$date-width: $line-height-4;
+$date-spacing: rem(4px);
 $calendar-padding: $component-spacing--xl;
-$calendar-width: ($date-width * 7) + ($calendar-padding * 2);
+$calendar-width: ($date-width * 7) + ($date-spacing * 6);
```

Comment:
This can be tight on a mobile screen, 304px as set width. Can we make this more dynamic?